src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/proteins-list-item-info/ProteinsListItemInfo.scss

```diff
@@ -60,3 +57,42 @@ $content-width: $gene_image_width;
   justify-content: center;
   margin-top: 40px;
 }
+.errorMessage{
+  color: $red;
+  margin-right:20px;
+  font-weight: $normal;
+}
+
+.xrefGroupChevron {
+  text-align: right;
+  padding-right: 2px;
+  cursor: pointer;
+  font-weight: $normal;
+  min-width: 50px;
+
+  .chevron {
+    margin-left: 10px;
+    margin-bottom: -1px;
+    height: 12px;
+    width: 12px;
+  
+  }
+  .close {
```

Comment:
Although we don't follow the BEM methodology in its entirety, some of its CSS conventions, such as identifying an element and a modifier, are useful, because they help with readability. In the original BEM syntax it would look like so: `.element_modifier`, or so: `element--modifier`; but we often write it like so: `elementModifier`.

In this case, the element is the chevron, and this additional class is modifying the behaviour of the chevron. We would probably name it something like `.chevronOpened` or `.chevronRotated`.

Reply:
used chevronUp/Down